win32_event_log/README.md

```diff
@@ -26,7 +26,25 @@ Both methods are configured in `win32_event_log.d/conf.yaml` in the `conf.d/` fo
 
 #### List Windows Event channels
 
-First, identify the Windows Event Log channels you want to monitor. To see a list of channels, run the following command in PowerShell:
+First, identify the Windows Event Log channels you want to monitor.
+
+Depending on collection method, the channel name can be used for the following configuration parameters:
+- Datadog Logs: `channel_path`
+- Datadog Events: `path`
+- Datadog Events (legacy): `log_file`
+
+<!-- xxx tabs xxx -->
+<!-- xxx tab "Event Viewer" xxx -->
+
+To find the channel name for an Event Log in Windows Event Viewer, open the Event Log Properties window and refer to the `Full Name` field. In the following example, the channel name is `Microsoft-Windows-Windows Defender/Operational`.
+
+<p align="center">
+<img alt="Windows Defender Operational Event Log properties" src="https://raw.githubusercontent.com/DataDog/integrations-core/master/win32_event_log/images/windows-defender-operational-event-log-properties.png"/>
+</p>
```

Comment:
```suggestion
{{< img src="https://raw.githubusercontent.com/DataDog/integrations-core/master/win32_event_log/images/windows-defender-operational-event-log-properties.png" alt="Windows Defender Operational Event Log properties" >}}
```

Reply:
The image is broken because I have to link it to master and the PR has not been merged into master yet, so the image does not exist yet.

Your suggestion did not work when I tried it locally. It produced an image with no source. Please refer to [this ticket](https://datadoghq.atlassian.net/browse/WEB-3066) which explains why I have to embed the image this way, instead of the way images are embedded in the [documentation repo](https://github.com/DataDog/documentation/wiki/Import-an-Image-or-a-mp4-video).